loqusdbapi/main.py

```diff
@@ -139,3 +143,32 @@ def read_case(case_id: str, db: MongoAdapter = Depends(database)):
             status_code=status.HTTP_404_NOT_FOUND, detail=f"Case {case_id} not found"
         )
     return case
+
+
+@app.delete("/cases/{case_id}")
+async def delete_case(
+    background_tasks: BackgroundTasks, case_id: str, db: MongoAdapter = Depends(database)
+):
+    existing_case = db.case({"case_id": case_id})
```

Comment:
```suggestion
    existing_case: models.Case = db.case({"case_id": case_id})
```

Reply:
db.case should be renamed to "db.get_case" or similar, but I am guessing that it out of scope for this PR

Reply:
This would have to be changed in the loqusdb repo.  Perhaps I can revisit that repo again and port the relevant code into this one eventually. Then we can rename things and get rid of cli dependencies when we're ready